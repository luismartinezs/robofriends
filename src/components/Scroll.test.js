import React from "react";
import { render, shallow } from "enzyme";
import Scroll from "./Scroll";

let wrapper;
beforeEach(() => {
  function LargeBox() {
    return <div style={{ height: "3000px", width: "100%" }} />;
  }
  wrapper = render(
    <Scroll>
      <LargeBox />
    </Scroll>
  );
});

it("expect to render children correctly", () => {
  expect(wrapper).toMatchSnapshot();
});
