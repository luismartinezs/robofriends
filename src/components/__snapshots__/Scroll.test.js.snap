// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render children correctly 1`] = `
<div
  style="overflow-y:scroll;border:5px solid black;height:800px"
>
  <div
    style="height:3000px;width:100%"
  />
</div>
`;
